rpc/pb/va/va.proto

```diff
@@ -0,0 +1,54 @@
+syntax = "proto2";
+
+package va;
+
+import "core/core.proto";
+
+service VA {
+	rpc IsSafeDomain(Domain) returns (Valid) {}
+	rpc PerformValidation(PerformValidationRequest) returns (ValidationRecords) {}
+}
+
+message Domain {
```

Comment:
If protobufs allow that, that seems fine for single-argument calls where the argument is a primitive type.